src/DependencyInjection/Configuration.php

```diff
@@ -214,4 +215,23 @@ private function createSmsConfiguration(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    private function createLocaleCookieConfiguration(ArrayNodeDefinition $root)
+    {
+        $root
+            ->children()
+                ->arrayNode('locale_cookie')
+                    ->info('Cookie settings for locale cookie')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('name')->end()
+                        ->scalarNode('domain')->end()
+                        ->integerNode('expire')->end()
+                        ->scalarNode('path')->end()
+                        ->booleanNode('secure')->end()
+                        ->booleanNode('httpOnly')->end()
```

Comment:
What's your opinion on leaving `secure` and `http_only` out and just enable those? Removes any risk of misconfiguration (security-wise).